internal/operator/operator.go

```diff
@@ -567,14 +570,26 @@ var (
 // is chosen based on the content of the componentNames map. Add friendly names to that map to include additional
 // component statuses to return.
 func (ic *InstallationClient) Status(ctx context.Context) ([]ComponentStatus, error) {
+	var err error
 	var installation operatorv1alpha1.Installation
 
 	const (
+		crdName  = "installations.operator.cert-manager.io"
```

Comment:
The name is `installations.operator.jetstack.io`

Reply:
oops, fixed in 1ba9f9b3ce2df61089c6b5d56632b2335d5ca131